tools/accuracy_checker/accuracy_checker/annotation_converters/antispoofing.py

```diff
@@ -0,0 +1,107 @@
+"""
+Copyright (c) 2018-2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from .format_converter import DirectoryBasedAnnotationConverter, ConverterReturn
+from ..representation import ClassificationAnnotation
+from ..utils import read_json, check_file_existence
+from ..config import PathField, NumberField
+
+class AntispoofingDatasetConverter(DirectoryBasedAnnotationConverter):
```

Comment:
DirectoryBasedAnnotationConverter is base class for converters which requires data_dir parameter. It used in cases when you have multiple files for getting annotation (e.g. annotation stored for every image independently) or gt files are images (e.g. in segmentation task).

FileBasedAnnotationConverter is base class for converters where all info stored in annotation file and it is enough for successful conversion (without advanced settings like labels overriding or file existence checking).

Both them were created for simplification parameters parsing for 2 most common used cases.

In case, when converter has a lot of custom parameter, you can use BaseFormat converter directly

So, on my view, file based converter description is closer to your converter flow than directory based